### DISCUSSION TITLE
More a question about templates.

### DISCUSSION THREAD

**Discussion by brimwats1** [argenos/zotero-mdnotes#48] [Q&A]
A question:

Is "Standalone Note" meant to be like "user defined template"?

Currently I use batch export because I want to mix and match fields, but I don't want thing repeated a bunch as tends to happen, so I had to to a lot of tweaking

 So for example my Default Template.md is 

```
{{title}}
{{citekey}}
{{related}}
{{collections}}
{{tags}}, #zotero-notes

***

##  Zotero links
{{localLibrary}}
{{cloudLibrary}}
{{pdfAttachments}}

***
```

Then the batch goes to my Metadata Template.md to get me 

```
## Bibliographic Data
{{itemType}}
{{author}}
{{proceedingsTitle}}
{{date}}
{{url}}
{{DOI}}
{{abstractNote}}

***

```

Then the batch goes to my Note Template.md to get me 

```
{{title}}
{{tags}}
{{related}}


{{noteContent}}

***

```

...could this all be added to "standalone note"? documentation isn't 100% clear so maybe "user defined template" to clear it up?

it would be great to be able to turn off all the other templates because I'll only be using my own user defined one

**Comment by argenos** [comment]
There are three main types of files I considered for mdnotes, each with their own templates:
1. A file/template for your own notes (mdnotes default file).
2. A file/template with the exported metadata.
3. A file/template with the contents of a Zotero note.

The standalone template is just an additional template that takes any of the placeholders of the default one,([see here](https://argentinaos.com/zotero-mdnotes/docs/templates/#standalone-note-template) for more details). An example of how I'm using this is to use the mdnotes default file as a hub for my own notes (a MOC that links to any other notes related to that particular reference), while the standalone notes are the atomic notes in my own words. That standalone template allows me to have an easy way to keep my notes linked to Zotero and prefill them with certain things from the metadata. 

As for what should be included in the templates, this depends on your settings a lot. It sounds to me like you're using the single-file setting? If that's the case, maybe take a look at the discussion in #44.

**Comment by brimwats1** [comment]
This question was asking if what I **should** be using is the single notes template. 

I follow this from the documentation:

```
Mdnotes Default Template Permalink
This template is used when you use the menu Create mdnotes file or when you have the single-file setting and choose Batch export, 
```

But as you can see in #47 i have had to modify each of the templates to make things look "clean"
